servers/physics_2d/godot_joints_2d.cpp

```diff
@@ -118,22 +118,25 @@ bool GodotPinJoint2D::setup(real_t p_step) {
 	K1[0].y = 0.0f;
 	K1[1].y = A->get_inv_mass() + B_inv_mass;
 
+	Vector2 r1 = rA - A->get_center_of_mass();
+	Vector2 r2 = B ? rB - B->get_center_of_mass() : rB;
```

Comment:
This seems used only in `if (B)`, so maybe it can move there and the ternary check can be removed.

Reply:
Indeed, thanks.